index.html

```diff
@@ -5,8 +5,62 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     </head>
-    <body>
-        <div>Hello World</div>
-        <script type="module" src="./src/script/index.js"></script>
+    <body hidden>
+        <div class="container__section header-container">
+            <header class="header">
+                <!-- Logo -->
+                <a href="/" class="header__logo" aria-label="home-link">
```

Comment:
```suggestion
                <a href="/" class="header__logo" aria-label="Go to homepage">
```

use descriptive labels